### ISSUE TITLE
Ability to set the UI language of Gaphor

### ISSUE THREAD

**Issue by sz332** [gaphor/gaphor#2586]
### Is your feature request related to a problem?

Gaphor uses the operating system's default language however very often it cannot be changed to english due company policy however I would like to use it in english.

The current solution is to set some kind of environment variable and start Gaphor but I always forget what it is, so it would be nice to have it configurable on the user interface somewhere.

### Describe the solution you'd like

This might be configurable in the initial dialog.

**Comment by sz332** [comment]
For the other: `set LANG=en_US.UTF-8` does the trick, but it is really not nice.

**Comment by tfirchau** [comment]
This issue seems to come up from time to time (see #1176, #2346, and #2461) and has also caused some friction for me. From the referenced issues I gather that @danyeaw and @amolenaar prefer the user interface language to be controlled by the operating system. 

> The current solution is to set some kind of environment variable and start Gaphor but I always forget what it is, so it would be nice to have it configurable on the user interface somewhere.

In #2470 I added a note on that in the documentation as kind of a compromise.

**Comment by danyeaw** [comment]
I also heard from someone at my company that his computer is locked by the version of Windows he has to Spanish, but he wants to use Gaphor in English. I hadn't considered that use case to be honest. What if instead of allowing a user to select a language, they have an option to override to English only.

In other words, they then by default get the system language, and the option to override to English as a checkbox option.

**Comment by sz332** [comment]
For a version 1 it would be totally fine by me. And if we still the keep relying on the `LANG` environmental variable then we are covering each and every case (maybe we can add a link to a website explaining how to set this variable for these use cases).

And if there is a need for language selection, we can implement it in a version 2. 

What do you think?

**Comment by tfirchau** [comment]
Why not make it exactly the same as the "Diagram Language" option and name it "Interface Language"? The selectable options could be the same set of mature translations. 

Then you can also use it if your system is locked to English, but you want Spanish, German, or whatever.

**Comment by tfirchau** [comment]
The Diagram Language setting is stored as part of the style sheet with the model. If you create a new model it is reset to the UI language.

The Show Tips setting is stored globally, for example. It keeps its state even if you create a new model. On one hand it would make sense to place the Interface Language setting next to that as it behaves in a similar way. On the other hand it would be weird to have the two parts of the language settings in two different places. Also, I kind of don't like that you can not navigate to the place where the Show Tips setting is explicitly. 

I think it would make more sense to place it in the preferences. But then it should also be clarified that some settings are "Model Preferences" and others are "Global Preferences".

**Comment by amolenaar** [comment]
Currently we do not have a global configuration file. I'm happy we didn't need to. Until now, it seems :wink:.

 The show tips thingy is stored per model. Not in the model, but in a per-model local settings file in `~/.config/gaphor`. The new (empty) model has a config file there as well.

I would go with @tfirchau's solution to create a menu with our supported languages. I think I would place it in the hamburger menu, not in the right-side Settings section, since you need to set it only once probably.
